src/com/oltpbenchmark/api/Worker.java

```diff
@@ -310,10 +318,14 @@ public final void run() {
                 }
             }
 
-            long startOperation = System.nanoTime();
+            MutableLong startOperation = new MutableLong();
+            MutableLong endOperation = new MutableLong();
+            MutableLong startConnection = new MutableLong();
+            MutableLong endConnection = new MutableLong();
```

Comment:
could we just pass in startConnection and endConnection, keep the old startOperation/endOperation computation outside here, and compute the difference if desired?

Reply:
I would prefer this, cuz the intention is clearer here. where we try to get the time of the actual operation execution.
Let me know what you think

Reply:
yeah i dont disagree. i might consider returning something richer than TransactionType that includes latency info as well, to further clarify/document the intention

Reply:
Thats a good idea. Let me incorporate that.